src/chromeDebug.ts

```diff
@@ -14,9 +16,10 @@ const EXTENSION_NAME = 'debugger-for-chrome';
 declare let VERSION: string;
 let versionWithDefault = typeof VERSION === 'undefined' ? 'unspecified' : VERSION; // Not built with webpack for tests
 
-// non-.txt file types can't be uploaded to github
-// also note that __dirname here is out/
-const logFilePath = path.resolve(__dirname, '../vscode-chrome-debug.txt');
+// non-.txt file types can't be uploaded to github. Generated filename is something like: vscode-chrome-debug_2017-07-19_161210.txt
+const timestampForFilename = new Date().toISOString().replace(/T/, '_').replace(/:/g, '').replace(/\..*/, '');
```

Comment:
Is this only needed for your tests? Could the adapter process be started with an environment var giving the name of the log file? Is that reasonable?

This is fine and tmpdir is fine, but I'm still thinking of a way to keep it writing to the same file on each run for vscode.

How will you find the log file? Are you just looking for the newest one in the tmpdir?